_src/css/_section.scss

```diff
@@ -0,0 +1,39 @@
+.section,.section--reverse{
+  display: flex;
+  justify-content: space-around;
+  height: 85vh;
+  background-color: lightblue;
```

Comment:
To make it better and lighter for the browser to read the rules, we can leave off the background-color and declare it in two separate elements: 

.section {
    background-color: lightblue;
}

.section--reverse {	
    flex-direction: row-reverse;
    background-color: lightpink
}